spec/paypal_express/base_plugin_spec.rb

```diff
@@ -9,10 +9,11 @@
       file = File.new(File.join(dir, 'paypal_express.yml'), 'w+')
       file.write(<<-eos)
 :paypal_express:
-  :signature: 'signature'
-  :login: 'login'
-  :password: 'password'
-  :test: true
+  - :account_id: paypal_test_account
```

Comment:
This change isn't strictly required, right? I just want to make sure the patch is backward compatible (i.e. when there is only one merchant account configured and the YAML value format for `:paypal_express` is a hash, not an array).